spec/models/project/storage_spec.rb

```diff
@@ -0,0 +1,112 @@
+#-- copyright
+# OpenProject is a project management system.
+# Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See doc/COPYRIGHT.rdoc for more details.
+#++
+
+require 'spec_helper'
+
+describe Project::Storage, type: :model do
+  let(:project1) { FactoryGirl.create(:project).reload }
+
+  let(:project2) { FactoryGirl.create(:project) }
+  let(:repository2) { FactoryGirl.create(:repository_git, project: project2) }
+
+  let(:wp) { FactoryGirl.create(:work_package, project: project1) }
+  let(:wikipage) { FactoryGirl.create(:wiki_page,
+                                       wiki: project1.wiki) }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 39, 61 should be on its own line.